client/packages/common/src/ui/components/navigation/AppNavLink/StyleFreeLink.tsx

```diff
@@ -0,0 +1,8 @@
+import Link, { LinkProps } from '@mui/material/Link';
+import { styled } from '@mui/material/styles';
+
+export const StyleFreeLink = styled(Link)<LinkProps>(({}) => ({
+  textDecoration: 'inherit',
+  color: 'inherit',
+  style: 'inherit',
+}));
```

Comment:
I think we should keep at least the underline-on-hover that we've used throughout (see for example the Breadcrumb links).

So you'd just add ` '&:hover': { textDecoration: 'underline' }` to this list.

You might want to then change the name of this component to "SimpleLink", or "UnstyledLink", or something like that 🤷‍♂️